build-tools/create-vsix/create-vsix.csproj

```diff
@@ -24,7 +24,6 @@
     <ExtensionInstallationFolder>Xamarin\Xamarin.Android.Sdk</ExtensionInstallationFolder>
     <OutputPath>bin\$(Configuration)</OutputPath>
   </PropertyGroup>
-  <Import Project="..\..\Configuration.props" />
```

Comment:
I believe you're removing the "wrong" `<Import/>`: you should keep this one -- in part because the convention used ~everywhere else is for the `.csproj` to import `Configuration.props`, not the `.targets` file, and also in part so that the `.csproj` can reliably use the contents of `Configuration.props` in the future -- and you should instead remove the `<Import/>` within `build-tools/installers/create-installers.targets`.